packages/web-components/src/components/va-date/va-date.tsx

```diff
@@ -267,7 +267,6 @@ export class VaDate {
               invalid={this.invalidMonth}
               class="select-month uswds-false"
               aria-label="Please enter two digits for the month"
-              uswds={false}
```

Comment:
I'm just curious why we needed to update `va-date`?

Reply:
We needed to update it because the stencil build was failing because the uswds prop was removed from va-select.